data/sources/blocks/e-commerce/use-of-cloud-computing-services-block-42.yaml

```diff
@@ -0,0 +1,13 @@
+id: use-of-cloud-computing-services-block-42
+title: Use of cloud computing services
+description: "<p>You have completed the previous section.</p><p>This section covers the use of cloud computing.</p>\
```

Comment:
Whenever you reach the interstitial page for "Use of cloud computing services" is that large spacing between the title and the description meant to be that much?

Reply:
Removed.